packages/footer/src/js/react.js

```diff
@@ -23,17 +23,19 @@ import PropTypes from 'prop-types';
  *
  * @param  {node}   children         - The inside of this section
  * @param  {string} className        - An additional class, optional
+ * @param  {string} ariaLabel        - The aria-label attribute, optional
  * @param  {object} attributeOptions - Any other attribute options
  */
-export const AUfooterNav = ({ children, className = '', ...attributeOptions }) => (
-	<nav className={`au-footer__navigation ${ className }`} { ...attributeOptions }>
+export const AUfooterNav = ({ children, className = '', ariaLabel = 'footer navigation', ...attributeOptions }) => (
+	<nav className={`au-footer__navigation ${ className }`} aria-label={ ariaLabel } { ...attributeOptions }>
```

Comment:
In react we need to use camel case `ariaLabel` like we do for `className`, have a look how this currently is outputted in the HTML.